pkg/kudoctl/cmd/install_test.go

```diff
@@ -23,3 +25,22 @@ func TestNewCmdInstallReturnsCmd(t *testing.T) {
 		t.Fatal("child and root commands should have the same normalization functions")
 	}
 }
+
+func TestNewCmdInstall_WithParameters(t *testing.T) {
+	newCmdInstall := NewInstallCmd()
+
+	newCmdInstall.Flags().Set("parameter", "foo")
```

Comment:
Consider switching to [table-driven tests](https://github.com/golang/go/wiki/TableDrivenTests) so we can add test cases more easily in the future.